File: backend/tests/fixtures/parsers/php/hello.php
<?php

namespace App;

use Other\Base;

class Greeter extends Base {
    public function greet($name) {
        $formatted = $this->formatName($name);
        echo $formatted;
    }

    private function formatName($name) {
        return strtoupper($name);
    }
}

function standaloneHelper() {
    $g = new Greeter();
    $g->greet("world");
}

File: backend/tests/fixtures/parsers/swift/hello.swift
import Foundation

class Greeter: Base {
    func greet(_ name: String) -> String {
        return formatName(name)
    }

    func formatName(_ n: String) -> String {
        return n.uppercased()
    }
}

func standaloneHelper() {
    let g = Greeter()
    _ = g.greet("world")
}
